src/pages/iou/WaypointEditor.js

```diff
@@ -143,6 +146,7 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
             const waypoint = {
                 lat: null,
                 lng: null,
+                name: waypointValue,
```

Comment:
Just a note that this note will make suggestions that are made offline not have the name field (currently there are both but both are same)

https://github.com/Expensify/App/assets/77237602/c1b31566-9535-4b54-887f-b66b57dafdba

Reply:
@neil-marcellini I think this is the only way the user is able to add a predefined place with no name... We should make sure we really want this to be the case because this is the only reason we would need to add fallbacks for the name field...